opencog/nlp/learn/scm/link-pipeline.scm

```diff
@@ -59,6 +59,9 @@
 ; as a clique of edges drawn between all words. This is explored further
 ; in the diary, in a section devoted to this topic.
 ;
+; Update: Additional counting modes can be requested by calling
+;         observe-text-mode. See function's documentation for details.
```

Comment:
Please remove this comment; it serves no purpose.

Reply:
OK. The word "Update" isn't needed. You could instead say, `Additional counting methods can be specified, including clique counting within a fixed window size. See the function description for additional modes.`

Reply:
Anyway, the clique-pair counting mode has always been there, from the very begining, and is already described by the docs. So this is not a new mode.  Also, I just now realized that this mode-setting style is not a good API; in the past, you could do both clique-counting, distance counting and "ANY" counting, all at the same time, with just one pass of the text.  With the new API, you would have to run the code three times, to get these three different modes.

You should probably think about some other API -- its a pointless waste of CPU time to run `observe-text` three difrerent times, to get three different counting modes, when running it just once is enough.

Reply:
This makes a lot of sense. I will try to implement it later on. Thanks!